include/gmsh.tcc

```diff
@@ -66,9 +64,13 @@ void GMSH::read_elements(const std::string& filename) {
 
   const unsigned toplines = 4;
   const unsigned ndimension = 3;
+  // specify element type 4 = tetrahedral
```

Comment:
For unstructured 2D it's triangles, and for unstructured 3D it's 4 node-tetrahedral.